bin/test_redirect

```diff
@@ -65,6 +65,7 @@ cat <<EOF > out/endpoint.json
 EOF
 
 base64=$(base64 -w 0 out/endpoint.json)
```

Comment:
Sigh this is similarly not OSX friendly. Can you comment OSX needs -b 0 for future archeologists?

Reply:
I really don't want to get into supporting OSX unless there's an official plan to do so -- e.g. proper QA and whatever behind it. It's just an unnecessary distraction for the core dev work we need to do.  If somebody wants to do OSX that's fine but then it needs to be on them to tweak the tools to work.